### ISSUE TITLE
Debt repayment description, amount auto-fill

### ISSUE THREAD

**Issue by johnjacobkenny** [AndreAle94/moneywallet#31]
I added a debt entry using the FAB on the **Debts** page. I can see the newly added debt on the same page. When I click on pay, it takes me to the **New transaction** page.

I would like it if the description and the amount are pre-filled using the value of the debt which I am paying. Is this supported currently? Thanks!

**Comment by AndreAle94** [comment]
It could be a good idea. The problem is that the application allows you to pay off a debt (credit) with multiple transactions. 
Example: the user adds a debt of € 50 and agrees to pay it in installments of € 10 per month; after the first month, clicking on "pay" he opens the page to add the transaction: if the application here pre-filled the fields with an amount of € 50 (and a related description as you requested) it would be wrong because the user would like to enter € 10 instead.

**Comment by johnjacobkenny** [comment]
Yes, you are right about that. So these are two cases, and depends on the user's preferences.

I think it would be useful to have a setting for this in case neither option is the more frequently used one.

In my opinion, having some value there by default is more useful than having it empty.

**Comment by gimiki** [comment]
Hello,
I want just to add one point: maybe is also good to have an option to add a "recurrence" to pay a dept, for example when there is an automatic withdrawal every month from your bank wallet.

Thank you,
Luigi
